index.js

```diff
@@ -132,8 +140,20 @@ module.exports = {
       if (isActions) {
         console.log('\n------------------Functions------------------\n');
         try {
+          if (inputs.envs.length > 0) {
+            const envs = {};
+            let envMessage = 'Forwarded the following environment variables: ';
+            inputs.envs.forEach((env, index) => {
+              envs[env] = process.env[env];
+              envMessage += index === inputs.envs.length - 1 ? env : env + ', ';
+            });
+            await writeFile('env.json', JSON.stringify(envs));
```

Comment:
Does this replace a file by that name if it exists?

Reply:
Updated.